elm.html.markdown

```diff
@@ -244,28 +244,28 @@ otherOrigin =
 -- But it's still the same type, so you can equate them.
 origin == otherOrigin -- True
 
--- By contrast, defining a union type creates a type that didn't exist before.
--- A union type is so called because it can be one of many possibilities.
--- Each of the possibilities is represented as a "tag".
+-- By contrast, defining a custom type creates a type that didn't exist before.
+-- A custom type is so called because it can be one of many possibilities.
+-- Each of the possibilities is represented as a "type variant".
 type Direction =
   North | South | East | West
 
--- Tags can carry other values of known type. This can work recursively.
+-- Type variants can carry other values of known type. This can work recursively.
 type IntTree =
   Leaf | Node Int IntTree IntTree
```

Comment:
hi!
sorry i don't speak elm. but your PR seems to be making valid changes and doing them well.
i just have a query with Ln.255
```
type IntTree =
	Leaf | Node Int IntTree IntTree
```
just checking. is the repetition of `IntTree` twice intentional?

Reply:
i did some more reading on the link you've attached. it's clearer now :+1: all good